### PR TITLE
PB-876 document settings

### PR DESCRIPTION
**Pull request by janpetschexain**
**References**
- [PB-876](https://xainag.atlassian.net/browse/PB-876)

**Summary**
- update settings documentation
- fix `lib.rs` intro text typos (as pointed out in #446 for the readme)

### CODE REVIEW

**Comment by janpetschexain** [comment]
> Looks good
> We should wait for #446 before merging this pr

sure, i'll update this accordingly.

File: rust/src/lib.rs [hunk 1]
```diff
@@ -52,15 +52,15 @@
...
 //! - It has a strong and static type discipline and traits, which describe shareable functionality
 //!   of a type.
 //! - It has rich functional abstractions, for example the `tower-service` based on the foundational
 //!   trait `Service`.
-//! - Its Idiomatic code compares favorably to Idiomatic C in performance.
-//! - It has no run-time and so is widely deployable. Foreign Function Interfaces support calls from
-//!   other languages, including Dart or Flutter.
-//! - And it compiles into LLVM, and so it can draw from the abundant tool suites for LLMV.
+//! - Its idiomatic code compares favorably to idiomatic C in performance.
+//! - It possible to compile without a run-time and so is widely deployable. Foreign Function
```
Comment (finiteprods) [inline rust/src/lib.rs]:
```suggestion
//! - It is possible to compile without a run-time and so is widely deployable. Foreign Function
```

File: rust/src/lib.rs [hunk 2]
```diff
@@ -9,7 +9,7 @@
 //! ## Making federated learning easy for developers
 //! Frameworks for machine learning - including those expressly for federated learning - exist
 //! already. These frameworks typically require the use of specific machine learning technology -
-//! for example tensorflow - or facilitate federated learning of cross-silo use cases - for
+//! for example Tensorflow - or facilitate federated learning of cross-silo use cases - for
```
Comment (finiteprods) [inline rust/src/lib.rs]:
```suggestion
//! for example TensorFlow - or facilitate federated learning of cross-silo use cases - for
```

File: rust/src/settings.rs [hunk 3]
```diff
@@ -207,7 +262,7 @@ pub struct MaskSettings {
     /// ```
     pub bound_type: BoundType,
 
-    /// The bounds of the numbers to be masked.
+    /// The maximum number of models to be aggregated at most.
```
Comment (finiteprods) [inline rust/src/settings.rs:265]:
```suggestion
    /// The maximum number of models to be aggregated.
```

File: rust/src/lib.rs (hunk 1 above)
Comment (janpetschexain) [reply rust/src/lib.rs]: updated from readme.

File: rust/src/settings.rs (hunk 3 above)
Comment (janpetschexain) [reply rust/src/settings.rs]: changed.

**Pull request merged**